Detections/ThreatIntelligenceIndicator/DomainEntity_imWebSession.yaml

```diff
@@ -0,0 +1,70 @@
+id: b1832f60-6c3d-4722-a0a5-3d564ee61a63
+name: TI map Domain entity to _Im_WebSession
+description: |
+  'Identifies a match in _Im_WebSession from any Domain IOC from TI'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ThreatIntelligence
+    dataTypes:
+      - ThreatIntelligenceIndicator
+queryFrequency: 1h
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+query: |
+    let dt_lookBack = 1h;
+    let ioc_lookBack = 14d;
+    //Create a list of TLDs in our threat feed for later validation
+    let list_tlds = ThreatIntelligenceIndicator
```

Comment:
As far as I recall, our analysis for DNS was that the tld calculation is an unneeded optimization. Probably not needed in the original, and just a waste in our version.